### PR TITLE
Issue #5021: Comments restyling + Gravatars as React component.

### PR DESCRIPTION
**Pull request by oksana-c**
Fixes issue [#5021](https://pm.savaslabs.com/issues/5021)

## Summary of changes

1. Adds `react-gravatar` package
1. Displays user avatars using md5 email hash. Default gravatar anonymous person is used for guests without gravatars. See here https://en.gravatar.com/site/implement/images/. Using provided default for now, will add a custom icon down the road.
1. Adds updated markup and styling for comments.

## Notes

this PR depends on https://github.com/savaslabs/squabble/pull/23

## To test

- [x] run Squabble server locally while on the branch linked above
- [x] Verify comment styling
- [x] submit comment using several different email addresses including the once you know have gravatars established (e.g. yours and my gmail)
- [x] verify gravatars are rendering immediately after the comment is submitted.

NOTE: I moved the comment form back to the bottom for the lack of time to properly adjust the animation on comment submit. When the form is at the top, submitted comment is displayed at the bottom of the page, but the won't see it because the page is not scrolled to the bottom. We should tweak this behavior down the road.

### Pull request reviewer

As the reviewer, I have verified the following:

- [x] I have tested the PR locally and/or in a staging environment

### CODE REVIEW

**Comment by AnneTee** [comment]
@oksana-c I added the `react-gravatar` library and removed `md5`, so please delete `node_modules` and run `npm install` then `gulp clean && gulp serve`. Even without the gravatars for tomorrow this is a great start - can't wait to see what it looks like when it's further along!

File: _assets/styles/scss/04-components/_comment.scss [hunk 1]
```diff
@@ -27,34 +27,68 @@ Markup is generated by Javascript by way of our <a href="https://github.com/sava
...
   }
 
-  .comment__text {
-    color: $dark-teal;
+  .comment__content--body {
+    padding: $base-spacing $base-spacing * 2 $base-spacing;
+
+    .comment__content--body__date {
+      color: lighten($grey, 30%);
+      font-size: .75em;
```
Comment (AnneTee) [inline _assets/styles/scss/04-components/_comment.scss]: @oksana-c I found this to be hard to read, can we bump it up to .9em?

**Review by AnneTee — APPROVED** [review]
@oksana-c I think this looks really excellent. I left one comment in the code, and there are two codeclimate issues that need to be resolved, but once those are fixed please feel free to merge.

I think it might be nice to bump up the size of the gravatars a little bit if possible; they're a little hard to see at their current size. Maybe I need some 👓 ? But we could do this in the future if we want to.

**Comment by oksana-c** [comment]
Hi @AnneTee, I'm not in favor of enlarging the gravatars, because they will take up more space on mobile and the last thing I'd want to do is add another block of CSS. Although I still did it 👓 . We can tweak later if wanted to. 
Also, scaled up the date.
Another @todo is to display the date in the short format as the mockup shows. If you have a quick code snippet at your disposal please add.

**Pull request merged**